.github/workflows/push.yml

```diff
@@ -51,24 +59,18 @@ jobs:
         run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
         id: extract_branch
 
-  dispatch:
-    runs-on: ubuntu-latest
+  deploy:
     needs:
       - build_and_push
       - branch_name
-    steps:
-      - uses: actions/checkout@v2
-      - name: Extract branch name
-        shell: bash
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
-        id: extract_branch
-      - name: Repository Dispatch
-        uses: peter-evans/repository-dispatch@v1
-        with:
-          token: ${{ secrets.REPO_ACCESS_TOKEN }}
-          repository: hpi-schul-cloud/dof_app_deploy
-          event-type: dev-deploy
-          client-payload: '{"branch": "${{ steps.extract_branch.outputs.branch}}" }'
+    uses: hpi-schul-cloud/dof_app_deploy/.github/workflows/deploy.yml@BC-589
```

Comment:
do not forget to change this after merge

Reply:
change back are done, will work if hpi-schul-cloud/dof_app_deploy#57 is merged